openwisp_radius/urls.py

```diff
@@ -16,7 +16,7 @@ def get_urls(api_views=None, social_views=None, saml_views=None):
     """
     return [
         path('api/v1/', include(get_api_urls(api_views))),
-        path('api/v1/', include(get_private_store_urls())),
+        path('', include(get_private_store_urls())),
```

Comment:
not sure I understood this? Can you explain why this is being changed?

Reply:
Okay, so `CSV_URL_PATH` variable is used for both the [API urls](https://github.com/openwisp/openwisp-radius/blob/6ea0fcd54b54d653678892c0d310412e064fb1cc/openwisp_radius/private_storage/urls.py#L12) and [admin dashboard urls](https://github.com/openwisp/openwisp-radius/blob/60a8e145da615fb45249d30520a6199b17df058c/openwisp_radius/base/models.py#L819).
If I add `/api/v1` here. The path in admin dashboard is `api/v1/<rest-of-the-path>` but the UI is pointing to `api/v1/api/v1/<rest-of-the-path>`.

Reply:
Can we use `reverse_lazy` here so that something like this never happens again? 

https://github.com/openwisp/openwisp-radius/blob/60a8e145da615fb45249d30520a6199b17df058c/openwisp_radius/base/models.py#L817-L820

Reply:
> Can we use `reverse_lazy` here so that something like this never happens again?

No, we cannot. I tried it locally.